student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom:, age:, parent_permission: true, name: 'Unknown')
+    super(age: age, name: name, parent_permission: parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯\(ツ)/¯'
```

Comment:
- [x] Kindly make sure you have set your class properly as you may notice how it's not possible to create an instance of your class
![image](https://github.com/emhamza/opp-school-library/assets/97100091/0e00ed6b-8a9b-4aee-b5a6-a72c936cd133)
```suggestion
class Student < Person
  attr_accessor :classroom

  def initialize(age, classroom, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @classroom = classroom
  end

  def play_hooky
    '¯\(ツ)/¯'
```

Reply:
For sure, thank you so much for your progressive feedback 🥇